store/tikv/region_cache.go

```diff
@@ -545,6 +545,11 @@ type RegionVerID struct {
 	ver     uint64
 }
 
+// GetID returns the id of the region
+func (r *RegionVerID) GetID() uint64 {
```

Comment:
Is this thread safe and does it need to be? @disksing

Reply:
I think it is safe here because it will never change. PTAL @disksing

Reply:
So `getID` is enough?

Reply:
Both fine.